packages/twenty-server/src/engine/core-modules/twenty-config/utils/convert-config-var-to-app-type.util.ts

```diff
@@ -0,0 +1,45 @@
+import { ConfigVariables } from 'src/engine/core-modules/twenty-config/config-variables';
+
+/**
+ * Convert a database value to the appropriate type for the environment variable
+ */
+export const convertConfigVarToAppType = <T extends keyof ConfigVariables>(
+  dbValue: unknown,
```

Comment:
these "unknown" + "as" you usage seem non stardard to me?
if dbValue is null, we know it's null but we cast it as ConfigVariables[T] ?